src/compiler/scala/tools/nsc/ast/parser/Scanners.scala

```diff
@@ -1379,7 +1378,7 @@ trait Scanners extends ScannersCommon {
     /** The source code with braces and line starts annotated with [NN] showing the index */
     private def markedSource = {
       val code   = unit.source.content
-      val braces = code.indices filter (idx => "{}\n" contains code(idx)) toSet
+      val braces = code.indices.filter(idx => "{}\n" contains code(idx)).toSet
       val mapped = code.indices map (idx => if (braces(idx)) s"${code(idx)}[$idx]" else "" + code(idx))
```

Comment:
missed this one?

Reply:
targeting postfix not infix per se on this trip.